forms/segnala_help_page.html

```diff
@@ -178,6 +178,23 @@
           'Zip Code': 'CAP',
           'I confirm to have read and accepted privacy informative to threat personal data': 'Confermo di aver letto e accetto l\'informativa sulla privacy per il trattamento dei dati personali',
           'Submit': 'Invia' 
+      },
+      ua: {
+        'Name/Surname' : 'Ім'я/Прізвище',
+        'Phone Number' : 'Номер телефону',
+        'phone number with international prefix': 'Номер телефону з міжнародним префіксом',
+        'What' : 'Що',
+        'Recipients': 'Одержувачі',
+        'Description': 'Опис',
+        'Location': 'Розташування',
+        'Address 1': 'Адреса 1',
+        'Address 2': 'Адреса 2',
+        'City': 'Місто',
+        'State': 'Область',
+        'Country': 'Країна',
+        'Zip Code': 'Почтовий індекс',
+        'I confirm to have read and accepted privacy informative to threat personal data': 'Я прочитав та згоден на обробку моїх персональних даних',
```

Comment:
```suggestion
        'I confirm to have read and accepted privacy informative to treat personal data': 'Я прочитав та згоден на обробку моїх персональних даних',
```

I assume it should be this way?

Reply:
Yes, thanks Vlad! Feel free to change the english lines too